<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background : rgb(255, 71, 9)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="NameLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>541</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../lab04.jpeg</pixmap>
    </property>
   </widget>
   <widget class="QTableWidget" name="TableBook">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>751</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(145, 125, 14);
color : rgb(255, 255, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="BackgroundLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(217, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(255, 71, 9);
border-radius : 15;

</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(255, 71, 9);
border-radius : 15;

</string>
    </property>
    <property name="text">
     <string>Delete book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(255, 71, 9);
border-radius : 15;

</string>
    </property>
    <property name="text">
     <string>Add book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SortButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(255, 71, 9);
border-radius : 15;

</string>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(255, 71, 9);
border-radius : 15;

</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InfoButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(255, 71, 9);
border-radius : 15;

</string>
    </property>
    <property name="text">
     <string>Information</string>
    </property>
   </widget>
   <widget class="QLabel" name="InfoLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : rgb(255, 255, 255);
background : rgb(217, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : rgb(255, 255, 255) ; 
color : rgb(255, 71, 9);
border-radius : 15;

</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
